src/google-ads-api.ts

```diff
@@ -255,6 +280,136 @@ export class GoogleAdsApi {
       ADGROUP_EXTENSION_SETTINGS: `
         SELECT ad_group_extension_setting.resource_name, ad_group_extension_setting.extension_feed_items, ad_group.id FROM ad_group_extension_setting WHERE ad_group_extension_setting.extension_type = 'IMAGE'
       `,
+      EXPERIMENT_ARMS: `
+        SELECT experiment_arm.campaigns FROM experiment_arm WHERE experiment.status = 'ENABLED'
+      `,
     };
   }
+
+  /**
+   * Returns a list of campaigns for which experiments can be created.
+   *
+   * @param campaignIds List of campaigns where experiments need to be created
+   */
+  filterOutCampaignsWithExperiments(campaignIds: string[]) {
+    const campaignsWithExperiments = this.executeSearch(
+      GoogleAdsApi.QUERIES.EXPERIMENT_ARMS
+    )
+      .map(e => e.experimentArm.campaigns)
+      .reduce(
+        (acc, value) =>
+          acc.concat(value.map((e: string) => e.split('/').slice(-1)).flat()),
+        []
+      );
+
+    return campaignIds.filter(e => !campaignsWithExperiments.includes(e));
+  }
+
+  /**
+   * Creates a partial experiment (without experiment arms)
+   *
+   * @param campaignId Campaign ID
+   */
+  createExperiment(campaignId: string) {
+    const experiment = {
+      // Name must be unique.
+      name: `Adios Experiment for campaignId ${campaignId} (timestamp:${Date.now()})`,
+      type: 'SEARCH_CUSTOM',
+      suffix: '[Adios experiment]',
```

Comment:
nit: Consistency with capitalization as in L316
```suggestion
      suffix: '[Adios Experiment]',
```